Formula/swiftgen.rb

```diff
@@ -2,18 +2,17 @@ class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
   url "https://github.com/SwiftGen/SwiftGen.git",
-      :tag      => "6.1.0",
-      :revision => "c6477a6950d313c94c964c6c8ec960e3aeccb8f6"
-  head "https://github.com/SwiftGen/SwiftGen.git"
+      :tag      => "6.2.0",
+      :revision => "c62c7ff1c86323b9c1e4c12bccfca19737dd2e56"
+  revision 1
+  head "https://github.com/SwiftGen/SwiftGen.git", :branch => "stable"
 
   bottle do
     cellar :any
-    sha256 "d3a00e098b7c4dd3e6add33e935976a52697a67384b0133b9c37e510f8ce4271" => :mojave
-    sha256 "786366406526b11d0958340597eb98e6e3d14971ca0a5097bfb11bae9a64cb03" => :high_sierra
```

Comment:
no need to touch the bottle block either.

Reply:
This was necessary because if I didn't remove this, when testing the formula locally before submitting my PR, homebrew tried to download the `mojave` bottle while I'm on macOS 10.15 and ended up with a 404 error that interrupted the test (instead of falling back to building from source). I couldn't test my formula locally without removing the bottles there. I suspect that's ok since once the PR is merged, homebrew's CI will recreate the bottles and amend the formula automatically to add them back anyway, right?

Reply:
Only if the bottle block is untouched. You can test next time by using the `-s` `--build-from-source` option: https://docs.brew.sh/Manpage#install-options-formula

Reply:
Fixed 👍